README.md

```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
 
 This will create a `.csv` file with information about the energy that you used to carry out the classification task, and an estimate of the CO<sub>2</sub> that you generated, complete with comparisons to common modes of transportation to give you a better idea of the order of magnitude of your emissions.
 
+# Report your emissions: LateX template
+
+We believe that an important step towards reducing carbon emissions is the generalization of emissions reporting in papers, blog posts and publications in general. Here's an example LateX snippet you might want to use:
+
+```latex
+\usepackage{hyperref}
+
+\subsection{CO2 Emission Related to Experiments}
+
+Experiments were conducted using {cloud provider} in region {region of compute}, which has a carbon efficiency of {carbon efficiency} kgCO$_2$eq/kWh. A cumulative of {hours used} hours of computation was performed on hardware of type {hardware type} (TDP of W).
+
+Total emissions are estimated to be {emission} kgCO$_2$eq of which {percentage offset} percents were directly offset by the cloud provider.
+
+%Uncomment if you bought additional offsets:
+%XX kg CO2eq were manually offset through \href{link}{Offset Provider}.
+
+Estimations were conducted using the \href{https://github.com/mlco2/codecarbon}{CodeCarbon emissions tracker}, a joint effort from authors of \cite{lacoste2019quantifying} and \cite{lottick2019nergy}.
+
+@article{lacoste2019quantifying,
+  title={Quantifying the Carbon Emissions of Machine Learning},
+  author={Lacoste, Alexandre and Luccioni, Alexandra and Schmidt, Victor and Dandres, Thomas},
+  journal={arXiv preprint arXiv:1910.09700},
+  year={2019}
+}
+
+@article{lottick2019nergy,
+    title={Energy Usage Reports: Environmental awareness as part of algorithmic
+  accountability},
+    author={Kadan Lottick and Silvia Susai and Sorelle A. Friedler and Jonathan P. Wilson},
+    year={2019},
+    journal={arXiv preprint arXiv:1911.08354}
+}
+```
+
+To find the carbon efficiency of your cloud region, you can look into [mlco2's data](https://github.com/mlco2/impact/blob/master/data/impact.csv). If you are using a private infrastructure, [a number of resources](https://github.com/mlco2/impact/tree/master/data#mlco2s-data) can help you find the carbon efficiency of you local grid.
```

Comment:
I think for the private infrastructure the right link is probably:
https://github.com/mlco2/codecarbon/tree/master/codecarbon/data/private_infra/2016

And similarly, it'd make sense to use the codecarbon link for the cloud regions too:
https://github.com/mlco2/codecarbon/tree/master/codecarbon/data/cloud

Reply:
Good points will do